data/categories.txt

```diff
@@ -1108,7 +1108,7 @@ sw:Vipodozi
 fa:لوازم ارایشی بهداشتی
 
 shop-convenience|@food|@shop
-en:4Convenience|U+1F3EC|U+1F3EA|grocer|grocery|mini-mart|mini-market|superette
+en:4Convenience|U+1F3EC|U+1F3EA|grocery|mini-mart|mini-market|superette
```

Comment:
grocer is not a typo, maybe leave it as is?

Reply:
Suggest will advise correct spelling, useless imo, or should add everywhere